Packs/CommunityCommonScripts/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Community Common Scripts",
     "description": "A pack that contains community scripts",
     "support": "community",
-    "currentVersion": "1.3.8",
+    "currentVersion": "1.4.0",
```

```suggestion
    "currentVersion": "1.3.9",
```
It seems that the change is not big and a ׳revision׳ is enough.